source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -429,6 +444,26 @@ DELETE /vms/vm-uuid/checkpoints/checkpoint-uuid/
 
 ### imageio backup API
 
+#### Disks backup_mode
+
+Before downloading the data, the user must check if incremental
+backup is available for a particular disk. If the system can provide
+incremental backup data for a disk, the disk `backup_mode` will be
+`incremetnal`. in this case the user can start an image transfer,
+download the backup using dirty extents to preform incremental backup.
+
+If the system cannot provide `incremetnal` backup data for a disk, the
+disk backup_mode will be `full`. The user can start an image
+transfer and download the backup using data and zero ranges to perform
+`full` backup of this disk. The next backup for this disk can be incremental.
+
+The system cannot provide incremental backup when:
+- adding a new disk to a VM
+- adding existing disk to a backup, which was not included in a
+  previous backup specified by the `<from_checkpoint_uuid>`
+- the system find that persistent dirty bitmaps are missing on
+  storage
+
```

Comment:
This is not related to imageio backup API. We don't have the concept of backup mode in imageio.